ang/civicase/PanelQuery.html

```diff
@@ -0,0 +1,56 @@
+<div class="panel panel-secondary">
+  <header class="panel-heading">
+    <div class="clearfix">
+      <h2 class="panel-title pull-left">{{title}}</h2>
```

Comment:
side note:

When attaching the `civicase-panel-query-title` using `$transcludeFn` you may want to create a new scope that extends from the parent scope and pass `$resultsCount` (or however you may want to call it) to it. I think this may fix any interference from other panels overwriting the same variable name.